hdr/sqlite_modern_cpp.h

```diff
@@ -399,41 +399,44 @@ namespace sqlite {
 		);
 	}
 	
+	enum class Encoding {
+	  AUTO = SQLITE_ANY,
+	  UTF8 = SQLITE_UTF8,
+	  UTF16 = SQLITE_UTF16
+	};
 	struct sqlite_config {
+    int flags = SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE;
```

Comment:
@zauguin Use an enum for flags instead of `int`.
```c++
enum class Flags {
  OPEN_READONLY = SQLITE_OPEN_READONLY,
  OPEN_READWRITE = SQLITE_OPEN_READWRITE,
  OPEN_CREATE = SQLITE_OPEN_CREATE,
  OPEN_NOMUTEX = SQLITE_OPEN_NOMUTEX,
  OPEN_FULLMUTEX = SQLITE_OPEN_FULLMUTEX,
  OPEN_SHAREDCACHE = SQLITE_OPEN_SHAREDCACHE,
  OPEN_PRIVATECACH = SQLITE_OPEN_PRIVATECACH,
  OPEN_URI = SQLITE_OPEN_URI
};
```